src/main/java/org/apache/rocketmq/flink/source/enumerator/RocketMQSourceEnumerator.java

```diff
@@ -337,16 +339,16 @@ private long getOffsetByMessageQueue(MessageQueue mq) throws MQClientException {
             } else {
                 switch (consumerOffsetMode) {
                     case CONSUMER_OFFSET_EARLIEST:
-                        offset = consumer.minOffset(mq);
-                        break;
+                        consumer.seekToBegin(mq);
+                        return -1;
                     case CONSUMER_OFFSET_LATEST:
-                        offset = consumer.maxOffset(mq);
-                        break;
+                        consumer.seekToEnd(mq);
+                        return -1;
```

Comment:
Return directly with -1, this is a different behavior compared to the old version or `CONSUMER_OFFSET_TIMESTAMP`.

It's recommended to unify the behavior.

Reply:
new api can not get the offset, when set maxoffset or minoffset